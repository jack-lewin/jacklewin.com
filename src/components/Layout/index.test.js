import React from 'react';
import renderer from 'react-test-renderer';
import { LayoutComponent } from '.';

describe('Layout', () => {
  const props = {
    children: [<p key={1}>some</p>, <p key={2}>content</p>],
  };

  const data = {
    site: {
      siteMetadata: {
        title: 'site title',
        description: 'site description',
      },
    },
  };

  it('should render as expected', () => {
    const element = renderer.create(<LayoutComponent data={data} {...props} />).toJSON();
    expect(element).toMatchSnapshot();
  });
});
