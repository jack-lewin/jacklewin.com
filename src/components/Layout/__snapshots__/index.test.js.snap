// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should render as expected 1`] = `
<div
  className="container"
>
  <p>
    some
  </p>
  <p>
    content
  </p>
</div>
`;
